client/scss/components/_workflow-tasks.scss

```diff
@@ -1,30 +1,30 @@
 .workflow-tasks {
-  $task-width: 117px;
-  $task-height: 56px;
+  $task-width: 7.313rem;
+  $task-height: 3.5rem;
```

Comment:
I’m surprised we’d want to do this in `rem`. Isn’t that going to break the layout?